custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnIssue.kt

```diff
@@ -37,6 +41,13 @@ class WorkOnIssue(
     private val logger: Logger = LogManager.getLogger(this::class.java)
 
     override fun run() {
+
+        val issueKeyMemoryIdsList: List<String> = getIssueKeyMemoryIds()
```

Comment:
Parsing properties is being executed for each action run
Please initialize the memory only once

Reply:
I've removed this code, we were trying to memorize the Issuekeys but it's not making much sense.